.github/workflows/ci.yaml

```diff
@@ -92,4 +92,8 @@ jobs:
     - run: make WHAT=test/e2e/e2e.test
       working-directory: ./kubernetes
     - run: WORKDIR=/tmp/e2e hack/run-sharded-kcp.sh
-    - run: WORKDIR=/tmp/cluster hack/run-cluster-kcp.sh
\ No newline at end of file
+    - run: WORKDIR=/tmp/cluster hack/run-cluster-kcp.sh
+    - uses: actions/upload-artifact@v2
+      with:
+        name: e2e-artifacts
+        path: /tmp/e2e/
```

Comment:
what will land there? binaries? logs?

Reply:
Logs and other test output goes here. Do we have a strong preference for "what" ?

Reply:
I guess right now it also includes the a couple directories which we likely don't need:

```
$ tree /tmp/e2e/
/tmp/e2e/
├── TestWorkspaceController_add_a_shard_after_a_workspace_is_unschedulable,_expect_it_to_be_scheduled
│   └── kcp
│       └── main
│           ├── admin.kubeconfig
│           ├── apiserver.crt
│           ├── apiserver.key
│           ├── kcp.log
│           ├── member
│           │   ├── snap
│           │   │   └── db
│           │   └── wal
│           │       ├── 0000000000000000-0000000000000000.wal
│           │       └── 0.tmp
│           └── secrets
│               ├── ca
│               │   ├── cert.pem
│               │   └── key.pem
│               ├── client
│               │   ├── cert.pem
│               │   └── key.pem
│               └── peer
│                   ├── cert.pem
│                   └── key.pem
```

Reply:
Right now I think the only real useful item is going to be the `*.log` items - I can filter down in a follow-up to direct test-driven data to a specific subset of this directory which we upload, while making sure that on a laptop the UX for looking at test artifacts like kubeconfig while a test runs is good